### PR TITLE
changes to readme for local setup

### PR DESCRIPTION
**Pull request by kliu28**

### CODE REVIEW

**Comment by cnk** [comment]
@kliu28 Could you please move the "install Postgres" item down into the "no containers" section? If you are running docker, the database will be a container managed with docker-compose rather than being on your machine.

File: README.md [hunk 1]
```diff
@@ -17,11 +17,12 @@ tools this project uses. (Don't spend too long on these; just use them to get a
 1. Clone this repository
 2. Get a copy of the initial data:
     - [Excel Sheet](https://drive.google.com/file/d/1xiLeyMwZc-n6eB1R_XdCJ00YrarfnR_w/view)
-    - Download as csv to the project root. In the steps below we refer to this file as `filename.csv`
+    - Download as csv to the project root. Make sure you choose the `CSV (Comma delimited) (*.csv)` option. In the steps below we refer to this file as `filename.csv`
+3. Install [PostgreSQL](https://www.postgresql.org/) if you haven't already.
```
Comment (cnk) [inline README.md:21]: This should be in the No Containers section. If using Docker, you do not need to install Postgres yourself.

**Review by cnk — CHANGES_REQUESTED** [review]

**Comment by kliu28** [comment]
updated. also edited numbering just for editing's sake; markdown technically doesn't care about the actual numbers in a numbered list

**Review by cnk — APPROVED** [review]
Thanks for these updates!

**Pull request merged**